src/unpatched-fonts/fontfilenames

```diff
@@ -3,6 +3,7 @@
 Agave/Agave-Regular.ttf
 AnonymousPro/Regular/Anonymous\ Pro.ttf
 Arimo/Regular/Arimo-Regular.ttf
+AtkinsonHyperlegibleMono/AtkinsonHyperlegibleMono-Regular.ttf
```

Comment:
Unrelated to the PR:

I must admit I have completely forgotten this file; would it not be for `git blame` I would have claimed I never saw it.

:thinking: 

Maybe I used it in ancient times, but nowadays I rip the Preview Source Font out of the `fonts.json` (programatically) I guess.

Probably drop it :thinking: